### DISCUSSION TITLE
Romanized Lyrics for Non-English Lyrics

### DISCUSSION THREAD

**Discussion by WilliamFong** [ciderapp/Cider#1081] [Feature Request]
I haven't seen any other apps have this feature, but I feel like it would be cool to have an option to enable romanized lyrics of non-english songs. I'm sure this will help with people who want to sing along to their favorite songs in another language.

**Comment by yazninja** [comment]
this works with musixmatch, just select your language + `romanji` e.g. `koran-romanji`

**Reply by WilliamFong** [reply]
I think the issue with this is that it is only specific to one language. If we listen to multiple languages, we will have to manually change. Not sure if there is a way to detect lyric languages and maybe auto-switch based on that.

But thank you for your response about the current functionality! I did not know that.
